packages/core/core-flows/src/cart/workflows/list-shipping-options-for-cart.ts

```diff
@@ -21,83 +21,103 @@ export const listShippingOptionsForCartWorkflow = createWorkflow(
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.service_zone_id",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.shipping_profile_id",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.provider_id",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.data",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.amount",
-
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.type.id",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.type.label",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.type.description",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.type.code",
-
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.provider.id",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.provider.is_enabled",
-
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.calculated_price.calculated_amount",
-        "stock_locations.fulfillment_sets.service_zones.shipping_options.calculated_price.is_calculated_price_tax_inclusive",
       ],
       variables: {
         id: input.sales_channel_id,
-        "stock_locations.fulfillment_sets.service_zones.shipping_options": {
-          context: {
-            is_return: "false",
-            enabled_in_store: "true",
-          },
-          filters: {
-            address: {
-              city: input.shipping_address?.city,
-              country_code: input.shipping_address?.country_code,
-              province_code: input.shipping_address?.province,
-            },
+      },
+    }).config({ name: "sales_channels-fulfillment-query" })
+
+    const fulfillmentSetIds = transform(
+      { options: scLocationFulfillmentSets },
+      (data) => {
+        const fulfillmentSetIds = new Set<string>()
+
+        deepFlatMap(
+          data.options,
+          "stock_locations.fulfillment_sets",
+          ({ fulfillment_sets }) => {
+            if (fulfillment_sets?.id) {
+              fulfillmentSetIds.add(fulfillment_sets.id)
+            }
+          }
+        )
+
+        return Array.from(fulfillmentSetIds)
+      }
+    )
+
+    const shippingOptions = useRemoteQueryStep({
+      entry_point: "shipping_options",
+      fields: [
+        "id",
+        "name",
+        "price_type",
+        "service_zone_id",
+        "shipping_profile_id",
+        "provider_id",
+        "data",
+        "amount",
+
+        "type.id",
+        "type.label",
+        "type.description",
+        "type.code",
+
+        "provider.id",
+        "provider.is_enabled",
+
+        "calculated_price.calculated_amount",
+        "calculated_price.is_calculated_price_tax_inclusive",
+      ],
+      variables: {
+        context: {
+          is_return: "false",
```

Comment:
we want to return options (that are enabled in store) as well on the storefront for customer returns
```suggestion
          is_return: "true",
```

Reply:
but I believe this should be an option in the query string no?
here it is the context, and if we set to `true` it will only bring the ones that are return.
I can remove it to return both.

Reply:
yes exactly we just need to remove `is_return` rule filter